### PR TITLE
Add WAFS jobs, scripts and ush to GFS v17 workflow repository

### PR DESCRIPTION
**Pull request by YaliMao-NOAA**
# Description
This PR adds WAFS jobs, scripts and ush to GFS v17 workflow repository

# Type of change

- New functionality

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO

# How has this been tested?

No testing. Simply copied the jobs/scripts/ush from the standalone WAFS package. EIB would modify one WAFS job to be adapted to the global workflow, and I will finish other jobs.

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -187,9 +175,6 @@ ush/global_cycle_driver.sh
 ush/jediinc2fv3.py
 ush/mkwfsgbl.sh
```
Comment (aerorahul) [inline .gitignore]: Is this a wafs script?

Comment (YaliMao-NOAA) [reply .gitignore]: It's not a wafs script.

Comment (YaliMao-NOAA) [inline .gitignore]: @aerorahul Sorry I forgot to remove this line: ush/mkwfsgbl.sh

**Review by aerorahul — APPROVED** [review]
looks good.
We will commit these as is and then work on each one as agreed.

**Pull request merged**